eng/scripts/Invoke-GeneratAndBuildV2.ps1

```diff
@@ -0,0 +1,87 @@
+#Requires -Version 7.0
+param (
+  [string]$inputJsonFile,
+  [string]$outputJsonFile
+)
+
+. (Join-Path $PSScriptRoot automation GenerateAndBuildLib.ps1)
+
+$inputJson = Get-Content $inputJsonFile | Out-String | ConvertFrom-Json
+$swaggerDir = $inputJson.specFolder
+$swaggerDir = Resolve-Path $swaggerDir
+$swaggerDir = $swaggerDir -replace "\\", "/"
+[string[]] $inputFilePaths = $inputJson.changedFiles;
+$readmeFiles = $inputJson.relatedReadmeMdFiles
+$commitid = $inputJson.headSha
+$repoHttpsUrl = $inputJson.repoHttpsUrl
+$downloadUrlPrefix = $inputJson.installInstructionInput.downloadUrlPrefix
+[string] $autorestConfig = $inputJson.autorestConfig
+
+$autorestConfigYaml = ""
+if ( $null -ne $autorestConfig -and $autorestConfig -ne "") {
+    $autorestConfig | Set-Content "config.md"
+    $autorestConfigYaml = Get-Content -Path .\config.md
```

Comment:
Use forward slashes in paths for cross platform support.
```suggestion
    $autorestConfigYaml = Get-Content -Path ./config.md
```

Reply:
updated